openedx/openedx-platform
# feat: add send survey report button

### Description:

This PR adds a new view to send the report to the external api and add button to send the report from django admin.

ISSUE:
https://github.com/openedx/tcril-engineering/issues/349

APPROVED DESIGN:
https://docs.google.com/document/d/1tm1O5BQsAwekphI5IN9pBn8behQDZigXePLGqCl5he8

### DECISIONS:
- We decided add a new view to manage the SurveyReportUpload actions.
- We decided add a send button for each report.

### How to test
With a dev environment running, you should go to the django admin /admin/survey_report/surveyreport/ and use the button "Generate Report" to generate a new survey report, then you could use the button send to send the report to the external API.

If the report has been already sent, the button message should be "Resend."
